ci/ci

```diff
@@ -279,13 +280,17 @@ if __name__ == "__main__":
     try:
         if args.configure != None:
             val = args.configure
+            write = True
             configure_opt = os.path.join(pbs_dirname, '.configure_opt')
             if val == 'called':
                 val = read_from_file(configure_opt)
                 if val == '':
                     val = '--prefix=/opt/pbs --enable-ptl'
+                else:
+                    write = False
             log_info("using configure parameters : " + val)
-            write_to_file(configure_opt, val)
+            if write:
+                write_to_file(configure_opt, val)
```

Comment:
why don't you just move the call to write_to_file() to lije 289 and do away with the else part and write variable?

Reply:
This is because val == 'called' comes into picture when no value is assigned to the option '--configure' but when there is some value assigned then like --configure='--prefix=/tmp/pbs --enable-ptl' then value in val will be the value assigned and we still need to update this value to the file.